Modules/Tags/src/Tags.Application/Tags/Queries/ListTags/ListTagsResponse.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+using Backbone.Modules.Tags.Domain;
+
+namespace Backbone.Modules.Tags.Application.Tags.Queries.ListTags;
+
+[JsonConverter(typeof(ListTagsResponseJsonConverter))]
+public class ListTagsResponse
+{
+    public required IEnumerable<string> SupportedLanguages { get; set; }
+    public required Dictionary<string, Dictionary<string, TagInfo>> TagsForAttributeValueTypes { get; set; }
+}
+
+public class ListTagsResponseJsonConverter : JsonConverter<ListTagsResponse>
+{
+    public override ListTagsResponse? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options) => null;
+
+    public override void Write(Utf8JsonWriter writer, ListTagsResponse value, JsonSerializerOptions options)
+    {
+        writer.WriteStartObject();
+
+        WriteSupportedLanguages(writer, value.SupportedLanguages);
+        WriteAttributes(writer, value.TagsForAttributeValueTypes);
+
+        writer.WriteEndObject();
+    }
+
+    private void WriteSupportedLanguages(Utf8JsonWriter writer, IEnumerable<string> supportedLanguages)
+    {
+        writer.WriteStartArray("supportedLanguages");
```

Comment:
You shouldn't hardcode the name:

```suggestion
        writer.WriteStartArray(nameof(ListTagsResponse).ToCamelCase());
```
(`ToCamelCase` has to be implemented by you)

Reply:
I just updated the suggestion to actually contain the property name instead of just `nameof(ListTagsResponse)`.